ddtrace/tracer/context_test.go

```diff
@@ -115,6 +116,7 @@ func Test128(t *testing.T) {
 	_, _, _, stop := startTestTracer(t)
 	defer stop()
 
+	os.Setenv("DD_TRACE_128_BIT_TRACEID_GENERATION_ENABLED", "false")
```

Comment:
Yeah, though I don't think it'll work in parallel anyway. I was having issues because `t.Setenv` doesn't run the `Cleanup` method to reset the env variable back to where it was before in a `t.Run`, at least from what I could see. That was causing issues for tests that have multiple `t.Run` calls but rely on different env values.